test/snapshot.ts

```diff
@@ -19,24 +19,25 @@ import * as assert from 'assert';
 import * as proxyquire from 'proxyquire';
 import * as sinon from 'sinon';
 
-import {PubSub, Subscription} from '../src';
-import {Snapshot} from '../src/snapshot';
+import {google} from '../proto/pubsub';
+import {PubSub, RequestCallback, RequestConfig, Subscription} from '../src';
 import * as util from '../src/util';
 
 let promisified = false;
 const fakePromisify = Object.assign({}, pfy, {
   // tslint:disable-next-line variable-name
-  promisifyAll(Class: Snapshot) {
+  promisifyAll(Class: Function) {
     if (Class.name === 'Snapshot') {
       promisified = true;
     }
   },
 });
 
 describe('Snapshot', () => {
-  // tslint:disable-next-line variable-name
-  let Snapshot;
-  let snapshot;
+  // tslint:disable-next-line variable-name no-any
```

Comment:
Instead of `any` here, can you do something like:
```js
import * as snapTypes from '../src/snapshot';
let Snapshot: typeof snapTypes.Snapshot:
let snapshot: Snapshot;
```

Note that `snapshot` shouldn't be `typeof` here